maintainers/maintainer-list.nix

```diff
@@ -15410,6 +15410,12 @@
     githubId = 3268082;
     name = "Thibaut Marty";
   };
+  thielema = {
+    name = "Henning Thielemann";
+    email = "nix@henning-thielemann.de";
+    github = "thielema";
+    githubId = 898989;
+  };
```

Comment:
@thielema it looks like you are not part of the @NixOS/nixpkgs-maintainers team, you likely missed the invite when you became maintainer / the team was created. This means you can't be requested for review properly. Please consider requesting another invite, see https://github.com/NixOS/nixpkgs/tree/master/maintainers#tools-for-maintainers for how to do that.